CoreFoundation/URL.subproj/CFURL.h

```diff
@@ -193,6 +193,9 @@ CFURLRef CFURLGetBaseURL(CFURLRef anURL);
  to request that none be.
  */
 
+CF_EXPORT
+CFURLRef CFGetStandardizedURL(CFURLRef url);
```

Comment:
lets not expose this as API, it should be SPI for foundation only.
Naming wise it should be _CFCopyStandardizedURL because it should return a retain count +1 from this method unless it is a direct getter.